internal/services/machinelearning/machine_learning_workspace_resource.go

```diff
@@ -141,6 +141,27 @@ func resourceMachineLearningWorkspace() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"encryption": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"key_vault_id": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: keyVaultValidate.VaultID,
+						},
+						"identifier": {
```

Comment:
Can we call this attribute 'key_id' to be consistent with other interfaces? There we use key_uri as we request an Azure resourceId but Terraform handles that better :)

Reply:
@denniseik  renamed to `key_id`